jukebox_radio/streams/models/queue_interval_model.py

```diff
@@ -99,8 +99,8 @@ def conflicting_interval(self, *, queue_id, lower_bound_id, upper_bound_id):
                 ) |
                 Q(
                     queue_id=queue_id,
-                    lower_bound=lower_bound,
-                    upper_bound=upper_bound,
+                    lower_bound__timestamp_ms=lower_bound.timestamp_ms,
```

Comment:
I probably shouldn't put _this_ bugfix in _this_ PR, but I'm moving fast and would rather just push forward.

Reply:
Fun stuff here: we cannot just compare FKs, we need to actually inspect the values to explicitly reject all completely overlapping intervals.